lib/spack/docs/tutorial_configuration.rst

```diff
@@ -732,6 +782,31 @@ preferences, we can specify version preferences as well. Anything
 that you can specify on the command line can be specified in
 ``packages.yaml`` with the exact same spec syntax.
 
+^^^^^^^^^^^^^^^^^^^^^^^^
+Installation Permissions
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``packages.yaml`` file also controls the default permissions
+to use when installing a package. You'll notice that by default,
+the installation prefix will be world readable but only user writable.
+
+Let's say we need to install ``converge``, a licensed software package.
+Since a specific research group, ``fluid_dynamics``, pays for this
+license, we want to ensure that only members of this group can access
+the software. We can do this like so:
+
+.. code-block:: yaml
+
+   packages:
+     converge:
+       permissions:
+         read: group
+         group: fluid_dynamics
+
+
+Now, the ``fluid_dynamics`` group has control over who can use any
```

Comment:
How about "Now, only members of the ``fuild_dynamics`` group can use any ``converge`` isntallations.